io_u.h

```diff
@@ -93,20 +93,26 @@ struct io_u {
 		struct workqueue_work work;
 	};
 
+	/*
+	 * ZBD Zone Append Support
+	 */
+	unsigned long long zone_slba;
```

Comment:
offset, not lba

Reply:
Will be renamed to zone_start_offset in the v2 patches.